cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -782,7 +783,7 @@ private boolean getProposals(HttpServletRequest request, HttpServletResponse res
       String loadAfterOptimization = optimizerResult.brokerStatsAfterOptimization().toString();
       if (!verbose) {
         out.write(
-            KafkaCruiseControlServletUtils.getProposalSummary(optimizerResult).getBytes(StandardCharsets.UTF_8));
+            KafkaCruiseControlServletUtils.getProposalSummary(optimizerResult, false).getBytes(StandardCharsets.UTF_8));
```

Comment:
If we pass `json` rather than an explicit `false` flag, would it make the code more clear?